### DISCUSSION TITLE
Welcome to Contextive Discussions!

### DISCUSSION THREAD

**Discussion by chrissimon-au** [dev-cycles/contextive#62] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by festnoze** [comment]
I'ld like to know if a version for Visual Studio 2022 will be released one day?
And if is it even possible with the technology you've implemented so far to port it to VS.

**Reply by chrissimon-au** [reply]
Hi @festnoze - yes, it is planned, hopefully sometime in '24.  The next IDE will be JetBrains based on overwhelming feedback in a recent survery, but VS 2022 would be the next after that. 

The implementation is as an 'independent language server' and it is written in .NET anyway, so it should be to port to VS 2022.

See https://github.com/dev-cycles/contextive/issues/28 to track updates on that work - as you'll see an initial poc highlight some issues with VS 2022 support for the 'language server', but I have some ideas on how to address that.

If you're interested in the technical implementation, you might also be interested in these Architecture Decision Records which address the thinking around implementation approaches to support multiple IDEs:

* https://github.com/dev-cycles/contextive/blob/main/docs/adr/0001-general-architecture.md
* https://github.com/dev-cycles/contextive/blob/main/docs/adr/0002-language-%26-framework-selection.md
